analysis/counts.py

```diff
@@ -20,15 +15,37 @@ def redact_small_numbers(df, column):
 )
 
 # choose only date variables
-activity_dates = df.filter(items=date_cols)
-activity_dates.columns = activity_dates.columns.str.replace("date_", "")
+activity_dates = df[[col for col in df.columns if col.endswith('_date')]]
+activity_dates.columns = activity_dates.columns.str.replace("_date", "")
 
 # count code activity per day
 codecounts_day = activity_dates.apply(lambda x: eventcountseries(event_dates=x, date_range = consec_dates))
 
+# select the codelists with multiple events 
+codelists = ["antigen_negative","exposure_to_disease","historic_covid","probable_covid_sequelae","potential_historic_covid","probable_covid", "probable_covid_pos_test","suspected_covid_advice","suspected_covid_had_test","suspected_covid_isolation","suspected_covid_nonspecific","suspected_covid","covid_unrelated_to_case_status","suspected_covid_had_antigen_test","sgss_positive_test"]
+
+#### help to decide on the appropriate maximum amount of events per patient. NOT TO BE RELEASED!!
+events_pp = pd.DataFrame(np.nan, index=range(1,n+1), columns=codelists)
+for list in codelists:
+    for i in range(1, n+1):
+        events_pp[list][i]=  codecounts_day[[col for col in codecounts_day.columns if col.startswith(f"{list}_{i}")]].sum(axis=0)
+
+events_pp.to_csv("output/events_pp.csv") # NOT TO BE RELEASED!!
```

Comment:
Am I allowed to create this table of the amount of patients with `n` number of events? It will not be released.